frontend/javascripts/oxalis/view/right-border-tabs/segments_tab/segment_statistics_modal.tsx

```diff
@@ -130,18 +131,30 @@ export function SegmentStatisticsModal({
     async () => {
       await api.tracing.save();
       if (requestUrl == null) return;
+      const maybeVolumeTracing =
+        tracingId != null ? getVolumeTracingById(tracing, tracingId) : null;
+      const maybeGetMappingName = () => {
+        if (maybeVolumeTracing?.mappingName != null) return maybeVolumeTracing?.mappingName;
```

Comment:
```suggestion
        if (maybeVolumeTracing?.mappingName != null) return maybeVolumeTracing.mappingName;
```

the second ? should not be necessary due to the first ?.